libqtile/xcbq.py

```diff
@@ -316,20 +316,20 @@ def alloc_color(self, color):
         """
             Flexible color allocation.
         """
-        if color.startswith("#"):
-            if len(color) != 7:
+        try:
+            return self.conn.conn.core.AllocNamedColor(
+                self.cid, len(color), color
+            ).reply()
+        except:
```

Comment:
Can we not use a bare except: here? It looks like all it can throw is xproto's BadColor, so we should limit the except clause to that.